src/TConsole.cpp

```diff
@@ -578,6 +578,7 @@ TConsole::TConsole(Host* pH, ConsoleType type, QWidget* parent)
         mDisplayFontSize = mDisplayFont.pointSize();
         refreshMiniConsole();
     }
+    setAcceptDrops(true);
```

Comment:
Is this adding the capability to ALL `TConsole`s - like the **Central Debug Console** or the **Error console** (in the bottom right corner of the editor) - obviously it won't affect **Buffers** as those do not have a physical on-screen representation so cannot be dropped on.  Is it intended for **User Windows** and what happens for a **SubConsole** - does the drop go to it or the `TConsole` instance it has been placed upon?

Reply:
Will add the capability only to UserWindows and to the MainConsole.
I assumed that if I drop on a subconsole in the mainwindow it won't work if this is not active.
But my assumptions were proven to be wrong on testing.